File: Greatest of 3 Numbers/Go2N.py
a=int(input("Enter 1st Number : "))
b=int(input("Enter 2nd Number : "))
c=int(input("Enter 3rd Number : "))
if a!=b and b!=c and a!=c:
    if a>b and a>c :
        print(a," is the Greatest Number ")
    elif b>a and b>c :
        print(b," is the Greatest Number ")
    else:
        print(c," is the Greatest Number ")
else:
    print("Try with 3 different Numbers")

File: Greatest of 3 Numbers/Go2N.cpp
#include<iostream>
using namespace std;
void main(){
    int a,b,c;
    cout<<"Enter 1st Number : ";
    cin>>a;
    cout<<"Enter 2nd Number : ";
    cin>>b;
    cout<<"Enter 3rd Number : ";
    cin>>c;
    if(a!=b && b!=c && a!=c){
        if(a>b && a>c){
            cout<<a<<" is the Greatest Number ";
        }else if(b>a && b>c){
            cout<<b<<" is the Greatest Number ";
        }else{
            cout<<c<<" is the Greatest Number ";
        }
    }else{
        cout<<"Try with 3 different Numbers";
    }
}